posts/frangeris/03.secure-values-by-private-keys-in-aws.md

```diff
@@ -0,0 +1,235 @@
+---
+title: Ensuring secure values by private keys in AWS (KMS, SSM, Secrets Manager)
+published: false
+description: Secure way of store and access sensible values on AWS
+tags: "aws, private keys security, serverless security, cloud security"
+cover_image: ./assets/03.cover.png
+---
+
+## Ensuring secure values by private keys in AWS (KMS, SSM, Secrets Manager)
+
+Private keys, credentials access and sensitive information such as passwords or any kind of confidential data are essentials in applications logic when interacting with other services or even databases, storing those values can be risky if proper security measures are not in place.
+
+In this post we will dig into one way to accomplish secure access to sensible information in AWS environments. For simplicity we will be using:
+
+- A ready to use [AWS account](https://aws.amazon.com/).
+- [Serverless Framework](https://www.serverless.com/framework/docs/getting-started) - to delegate the CloudFormation creation and deployment.
+
+> This implementation focuses on using specifics services and functionalities provided by AWS services. Implementations will vary depending on the provider (Azure, GCP).
+
+### Let's start by creating a basic [AWS Lambda](https://aws.amazon.com/lambda/)
+
+This function will try to access our "secure" and use it. By using `npx` command we get access to serverless executable, select the basic to keep it simple, there are options for different types of uses cases, even different languages options in case `Node.js` is not your language:
```

Comment:
```suggestion
This function will try to access our "secure" and use it. By using `npx` command we get access to serverless executable. I recommend you select the basic settings to keep it simple, but don't forger there are options for different types of uses cases, even different languages options in case `Node.js` is not your language.
```

Reply:
The suggestion has a typo

Reply:
Fixed